unfccc.csl

```diff
@@ -0,0 +1,243 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="in-text" demote-non-dropping-particle="sort-only" default-locale="en-GB" page-range-format="expanded">
+  <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>UNFCCC</title>
+    <title-short>UNFCCC</title-short>
+    <id>http://www.zotero.org/styles/unfccc</id>
+    <link rel="self" href="http://www.zotero.org/styles/unfccc"/>
+    <link href="https://www.zotero.org/styles/harvard-kings-college-london" rel="template"/>
+    <author>
+      <name/>
```

Comment:
@damnation333, we can typically contact contributors directly via GitHub. The point of adding your email address in the style is to make yourself available to user queries (if you're interested in answering those), but it comes at the risk of having your email address made public.